ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -69,11 +69,6 @@ is_chown_forbidden() {
   return $?
 }
 
-is_vboxsf_mountpoint() {
-  grep -q "/app vboxsf" /proc/mounts
-  return $?
-}
-
```

Comment:
we're just considering whether our original preferred generic fix (setfacl) would solve this without the need for an ext4 APP_USER_LOCAL